packages/core/lib/command-utils.js

```diff
@@ -25,6 +25,9 @@ const getCommand = ({ inputStrings, options, noAliases }) => {
   const firstInputString = inputStrings[0];
   let chosenCommand = null;
 
+  if (firstInputString === "-v" || firstInputString === "--version") {
```

Comment:
I guess this works but I think it is more clear if this is part of the main `if/else` below.

Reply:
Ah I see. Thank you for taking the time to explain this. I was keeping this outside thinking too many if and else if's can be confusing, but I do follow your logic since it is all checking the input to figure out the user's desired command.

Reply:
Yeah, my idea is kind of that they seem much more connected if they are all part of the same one. Then it's more like, "ok, either this, this, or this happens". :)